components/Downloads/Release/PlatformDropdown.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import type { FC } from 'react';
+import { useMemo } from 'react';
+
+import Select from '@/components/Common/Select';
+import Generic from '@/components/Icons/Platform/Generic';
+import Homebrew from '@/components/Icons/Platform/Homebrew';
+import NVM from '@/components/Icons/Platform/NVM';
+import { useReleaseContext } from '@/providers/releaseProvider';
+
+const PlatformDropdown: FC = () => {
+  const {
+    state: { platform },
+    dispatch: { setPlatform },
+  } = useReleaseContext();
+
+  const items = useMemo(
```

Comment:
This doesn't need to be mmemoized, nor needs to be inside a component. Feel free to assign to a constants file like the other one before.